app.json

```diff
@@ -2,11 +2,20 @@
   "name": "kingsmen-ah-backend",
   "description": "A community of like minded authors that fosters inspiration and innovation by leveraging the modern web.",
   "scripts": {},
-  "env": {},
+  "env": {
+    "SALT": {
```

Comment:
include the following env
```json
"HEROKU_APP_NAME": {
      "required": true
    },
    "HEROKU_PARENT_APP_NAME": {
      "required": true
    }
```

Reply:
Will do. I am guessing I will have to set those in Heroku environment too?

Reply:
Ok I understand now. Thanks